docs/registry/registry.md

```diff
@@ -0,0 +1,16 @@
+# Registries
+
+A registry persists metadata pertaining to DNS records.
+
+The most important metadata is the owning external-dns deployment.
+This is specified using the `--txt-owner-id` flag, specifying a value unique to the
+deployment of external-dns and which doesn't change for the lifetime of the deployment.
+Deployments in different clusters but sharing a DNS zone need to use different owner IDs.
```

Comment:
I understand the need to use different owner IDs but is there any logic that would enforce this?

Reply:
There's no way to cause an error message. Any mechanism to detect an fail on duplicate owner IDs would be equivalent to generating unique owner IDs.

Reply:
@johngmyers good point, thanks